tests/unit/containers/edit-profile/notification-tab/NotificationTab.spec.jsx

```diff
@@ -27,4 +27,9 @@ describe('NotificationTab', () => {
       expect(screen.getByText(item.title)).toBeInTheDocument()
     })
   })
+  it('renders update button', () => {
+    expect(
+      screen.getByText('editProfilePage.profile.updateProfileBtn')
```

Comment:
I suggest moving `screen.getByText('editProfilePage.profile.updateProfileBtn')` to constant

Reply:
Thank you. Fixed